util/topgen.py

```diff
@@ -50,6 +50,7 @@ def generate_xbars(top, out_path):
         xbar_path = out_path / 'ip/xbar_{}/data/autogen'.format(obj["name"])
         xbar_path.mkdir(parents=True, exist_ok=True)
         xbar = tlgen.validate(obj)
+        xbar.top_name = "top_" + top["name"]
```

Comment:
```suggestion
        xbar.ip_path = "hw/top_" + top["name"] + "/ip"
```

Reply:
Thanks @eunchan for the suggestion.
As discussed, change it to ip_path and make sure tlgen.py can generate xbar directly. PTAL